### DISCUSSION TITLE
Next Release? Release Candidate?

### DISCUSSION THREAD

**Discussion by codescalar** [edgedb/edgedb-js#872] [General] [answered]
There have been a number of performance improvements in regards to Typescript and the LSP in recent months. As an NX monorepo user with horrible LSP/TS performance due to edgedb types, I'm desperately awaiting the release of these improvements.

Any chance of cutting a release candidate? What should I expect in terms of timeline for these getting released?

#693 
#836
#832
#831
#830
#829

**Comment by scotttrinh** [comment, marked as answer]
@codescalar 

Apologies for just now getting back to you on this, I don't have notifications set up well for Discussions (I've fixed that now 😅 )

I plan on having a release in the next few weeks, but we actually publish a canary version on every push to `master`, so feel free to grab the latest one and check out any of those changes.

**Reply by codescalar** [reply]
@scotttrinh 

> I plan on having a release in the next few weeks

Love to hear it!

> we actually publish a canary version on every push to master

I'll definitely look into this, thanks!

FWIW I love using EdgeDB and the DX is great although there are still some gaps which bite me from time to time. Nothing that can't be sorted out with time and effort though! Would love to see future releases focused around performance and ops.

Keep up the great work!!
